### PR TITLE
zh-trans: /docs/contribute/style/write-new-topic.md

### PR DESCRIPTION
**Pull request by liyuan198251**
Signed-off-by: liyuan198251 <li.yuan4@zte.com.cn>

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.14 Features: set Milestone to 1.14 and Base Branch to dev-1.14
> 
> For Chinese localization, base branch to release-1.12
>
> For Korean Localization: set Base Branch to dev-1.13-ko.<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/docs/contribute/style/write-new-topic.md [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+本页面展示如何为 Kubernetes 文档库创建新主题。
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+<!--
+Create a fork of the Kubernetes documentation repository as described in
+[Start contributing](/docs/contribute/start/).
+-->
+
+创建 Kubernetes 文档仓库的一个克隆，如[开始贡献](/docs/contribute/start/)中所述。
```
Comment (xichengliudui) [inline content/zh/docs/contribute/style/write-new-topic.md]: 一个分支

File: content/zh/docs/contribute/style/write-new-topic.md [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+## Choosing a page type
+-->
+
+## 选择页面类型
+
+<!--
+As you prepare to write a new topic, think about the page type that would fit your content the best:
+-->
+
+准备撰写新主题时，请考虑最适合你内容的页面类型：
```
Comment (xichengliudui) [inline content/zh/docs/contribute/style/write-new-topic.md]: 当你准备写一个新的话题时，考虑一下最适合你的内容的页面类型:

File: content/zh/docs/contribute/style/write-new-topic.md [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+在你的主题中，在[页面头部](https://jekyllrb.com/docs/frontmatter/)设置一个 `title` 字段。页面头部是位于页面顶部三条虚线之间的 YAML 块。下面是一个例子：
+
+<!--
+    ---
+    title: Using an HTTP Proxy to Access the Kubernetes API
+    ---
+-->
+
+    ---
+    标题: 使用 HTTP 代理访问 Kubernetes API
```
Comment (xichengliudui) [inline content/zh/docs/contribute/style/write-new-topic.md]: `title` 不应该被翻译

File: content/zh/docs/contribute/style/write-new-topic.md [hunk 4]
```diff
@@ -0,0 +1,348 @@
...
+Choose a title that has the keywords you want search engines to find.
+Create a filename that uses the words in your title separated by hyphens.
+For example, the topic with title
+[Using an HTTP Proxy to Access the Kubernetes API](/docs/tasks/access-kubernetes-api/http-proxy-access-api/)
+has filename `http-proxy-access-api.md`. You don't need to put
+"kubernetes" in the filename, because "kubernetes" is already in the
+URL for the topic, for example:
+-->
+
+选择一个标题，标题中包含了要通过搜索引擎要查找的关键字。创建一个文件名，使用标题中由连字符分隔的单词。例如，标题为[使用 HTTP 代理访问 Kubernetes API](/docs/tasks/access-kubernetes-api/http-proxy-access-api/)的主题的文件名为 `http-proxy-access-api.md`。你不需要在文件名中加上 "kubernetes"，因为 "kubernetes" 已经在主题的 URL 中了，例如：
```
Comment (xichengliudui) [inline content/zh/docs/contribute/style/write-new-topic.md]:
```suggestion
选择一个标题，标题中包含了要通过搜索引擎要查找的关键字。创建一个文件名，使用标题中由连字符分隔的单词。例如，标题为[使用 HTTP 代理访问 Kubernetes API](/docs/tasks/access-kubernetes-api/http-proxy-access-api/) 的主题的文件名为 `http-proxy-access-api.md`。你不需要在文件名中加上 "kubernetes"，因为 "kubernetes" 已经在主题的 URL 中了，例如：
```

**Comment by xichengliudui** [comment]
/lgtm
/approve
thanks!

File: content/zh/docs/contribute/style/write-new-topic.md (hunk 1 above)
Comment (tengqm) [reply content/zh/docs/contribute/style/write-new-topic.md]: No. "fork" 是一个克隆。分支是 git 的 branch。

**Pull request merged**